tools/functional-tester/etcd-tester/tester.go

```diff
@@ -83,6 +91,33 @@ func (tt *tester) runLoop() {
 	plog.Printf("%s functional-tester is finished", tt.logPrefix())
 }
 
+func (tt *tester) setupStressers() {
+	tt.Stressers = make([]Stresser, len(tt.cluster.Members))
```

Comment:
Stetup stresser also starts stersser but setup checker does not invoke checker. Seems inconsistent. Can we fix this?